SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -123,6 +123,10 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(config, meta = (ConfigRestartRequired = false))
 	bool bCheckRPCOrder;
 
+	/** Seconds to wait before executing an RPC substituting null values for unresoled parameters */
+	UPROPERTY(config, meta = (ConfigRestartRequired = false))
+	float SecondsToProcessRPCWithUnresolvedRefs;
```

Comment:
Bit of a mouthful - but I think I prefer `WaitTimeBeforeProcessingReceivedRPCWithUnresolveRefs`

Reply:
It's already a mouthful, so I'll change as you suggest :)

Reply:
Updated.